packages/gitbeaker-cli/src/cli.ts

```diff
@@ -200,7 +200,7 @@ cli.command('*', (argv, ctx) => {
 });
 
 // Add all supported API's
-Object.entries(map).forEach(([apiName, methods]) => {
+Object.entries(getAPIMap()).forEach(([apiName, methods]) => {
```

Comment:
As opposed to using the `getAPIMap` function, what if we just had here the `__apiMap__` constant that contains already valid & parsed json as a javascript object to achieve the best performance?

```suggestion
Object.entries(__apiMap__).forEach(([apiName, methods]) => {
```

Reply:
See my previous points